src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard.ts

```diff
@@ -0,0 +1,119 @@
+}
+
+export const GCPBillingProjectWizard = ({ onSuccess, billingAccounts, authorizeAndLoadAccounts }: GCPBillingProjectWizardProps) => {
+  const persistenceId = 'billing'
+  const [accessToBillingAccount, setAccessToBillingAccount] = useState<boolean>(() => getLocalPref(persistenceId)?.accessToBillingAccount)
+  const [accessToAddBillingAccountUser, setAccessToAddBillingAccountUser] = useState<boolean | undefined>(() => getLocalPref(persistenceId)?.accessToAddBillingAccountUser)
+  const [verifiedUsersAdded, setVerifiedUsersAdded] = useState<boolean | undefined>(false) // if no access to add billing account user, has the user verified access
+  const [refreshed, setRefreshed] = useState<boolean>(false)
+  const [activeStep, setActiveStep] = useState<number>((getLocalPref(persistenceId)?.activeStep || 1))
+
+  useEffect(() => {
+    setLocalPref(persistenceId, { activeStep, accessToBillingAccount, accessToAddBillingAccountUser })
+  }, [persistenceId, activeStep, accessToBillingAccount, accessToAddBillingAccountUser])
+
+
+  return h(StepWizard, {
+    title: 'Link a Google Cloud billing account to Terra',
+    intro: `The linked billing account is required to cover all Google Cloud data storage, compute and egress costs incurred in a Terra workspace.
+        Cloud costs are billed directly from Google and passed through Terra billing projects with no markup.`,
+  }, [
+    h(GoToGCPConsoleStep, { isActive: activeStep === 1, stepFinished: () => setActiveStep(2) }),
+    h(BillingAccountAccessStep, {
+      accessToBillingAccount,
+      setAccessToBillingAccount: (access: boolean) => {
+        setAccessToBillingAccount(access)
+        setAccessToAddBillingAccountUser(undefined)
+        setVerifiedUsersAdded(undefined) // reset verification of step 3
+        setRefreshed(false) // this might or might not be necessary
+        setActiveStep(3)
+      },
+      isActive: activeStep === 2,
+    }),
+    // show AddTerraAsBillingAccountUserStep by default, and when the user selects that they have a billing account in step 2
+    // show ContactAccountAdminToAddUserStep if the user selects that they don't have a billing account in step 2
+    //     and transition to ContactAccountAdminToAddUserStep if the user selects that they don't have access to add a user in AddTerraAsBillingAccountUserStep
+    // This may be better to combine into a separate component - then `refreshed` states in step 3 and 4 could be separate
+    accessToBillingAccount !== false && accessToAddBillingAccountUser !== false ?
+      h(AddTerraAsBillingAccountUserStep, {
+        accessToAddBillingAccountUser,
+        setAccessToAddBillingAccountUser: async hasUserAccess => {
+          setAccessToAddBillingAccountUser(hasUserAccess)
+          if (hasUserAccess) {
+            await authorizeAndLoadAccounts()
+            setActiveStep(4)
+            setRefreshed(true)
+          } else {
+            setActiveStep(3)
+            //await authorizeAndLoadAccounts()
```

Comment:
Why is line 67 here? Did it used to call that?

Reply:
It's leftover from when I was teasing apart logic.